pkg/cmd/bootstrap.go

```diff
@@ -13,8 +13,10 @@ import (
 	"github.com/redhat-developer/kam/pkg/cmd/utility"
 	"github.com/redhat-developer/kam/pkg/pipelines"
 	"github.com/redhat-developer/kam/pkg/pipelines/ioutils"
+	"github.com/redhat-developer/kam/pkg/pipelines/statustracker"
 	"github.com/spf13/cobra"
 
+	err "github.com/pkg/errors"
```

Comment:
Not sure if we want to use "err" as an aliase because we have "err" as  variable name all over the place.

Reply:
It's also not clear why this is importing `pkg/errors` at all?

`pkg/errors` is deprecated as most of the functionality was folded into Go.

Reply:
+1

```
import "errors"

err := errors.New(" ...")
```